src/main/java/com/amazon/ion/impl/IonTypeID.java

```diff
@@ -286,8 +285,10 @@ private IonTypeID(byte id, int minorVersion) {
                     macroId = id;
                     length = 0;
                 } else {
-                    if (upperNibble == 0xF) {
-                        // FlexUInt length-prefixed macro invocation.
+                    if (id == E_EXPRESSION_FLEX_UINT) {
+                        variableLength = false;
+                        length = 1;
```

Comment:
What does `E_EXPRESSION_FLEX_UINT` indicate? A `FlexUInt`-encoded macro address? If so, I infer that `length` is actually something like `opcodeAndAddressLength`. If that's true, could we make the name more explicit?

Reply:
I originally wanted to change it, but I left the name as it was to avoid polluting the PR with (what I thought was) unnecessary renaming. I'll change the name to `E_EXPRESSION_WITH_FLEX_UINT_ADDRESS`.